monawe/urls.py

```diff
@@ -17,5 +17,5 @@
 
 urlpatterns = [
     path('user/', include('user.urls')),
-    path('product/', include('product.urls')),
+    path('order/', include('order.urls'))
```

Comment:
```suggestion
    path('order', include('order.urls'))
```
main url에는 슬래시('/')를 붙이지 않고, app url의 앞부분에 슬래시를 붙이시는 것이 위코드 규칙입니다 :)